common/culture/cultures/dwarf.txt

```diff
@@ -404,6 +404,7 @@
 	wildhammer = {
```

Comment:
Will you add markings to Falstad and Kurdran?

Reply:
I think I would prefer to do that on `dna-5`.